### PR TITLE
Fix GasConsumedMigrationTest

### LINKED ISSUE #8248: Flaky GasConsumedMigrationTest
### Description

```
GasConsumedMigrationTest > migrate() FAILED
    org.opentest4j.AssertionFailedError at GasConsumedMigrationTest.java:106
```

### Steps to reproduce

`./gradlew build` in CI

### Additional context

https://github.com/hashgraph/hedera-mirror-node/actions/runs/8990072383/job/24696404490?pr=8246

### Hedera network

other

### Version

main

### Operating system

None

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR fixes the GasConsumedMigrationTest

- Use non-zero byte array as contract result function parameters

**Related issue(s)**:

Fixes #8248 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/GasConsumedMigrationTest.java [hunk 1]
```diff
@@ -184,7 +184,7 @@ public static MigrationContractResult createMigrationContractResult(
                 .createdContractIds(List.of(domainBuilder.entityId().getId()))
                 .errorMessage("")
                 .failedInitcode(failedInitcode)
-                .functionParameters(domainBuilder.bytes(64))
+                .functionParameters(domainBuilder.nonZeroBytes(64))
```
Comment (xin-hedera) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/GasConsumedMigrationTest.java:187]: same as the fix for the other flaky gas consumed integration test, need to make sure there is no 0 byte in function parameters

**Review by jnels124 — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by mgoelswirlds — APPROVED** [review]
LGTM

**Pull request merged**